admin/views/about.php

```diff
@@ -270,25 +213,49 @@ function wpseo_display_contributors( $contributors ) {
 			wpseo_display_contributors( $leaders );
 			?>
 		</ul>
-		<h4 class="wp-people-group"><?php _e( 'Contributing Developers', 'wordpress-seo' ); ?></h4>
-		<ul class="wp-people-group " id="wp-people-group-core-developers">
+		<h4 class="wp-people-group"><?php _e( 'Recent Rockstars', 'wordpress-seo' ); ?></h4>
+		<ul class="wp-people-group " id="wp-people-group-rockstars">
 			<?php
 			$contributors = array(
```

Comment:
Can this array be held in a separate function? Can then be potentially re-used. Also, keeps data away from template.